docs/modules/user-guide/partials/proc_adding-a-registry-schema.adoc

```diff
@@ -0,0 +1,423 @@
+[id="adding-a-registry-schema_{context}"]
+= Adding a registry schema
+
+[role="_abstract"]
+To add samples and stacks from other repositories, put the file, `extraDevfileEntries.yaml`, under the root of a devfile registry. This file contains the location information of the extra samples and stacks.
+
+.extraDevfileEntries.yaml schema
+* samples: schema[] - the sample list
+* stacks: schema[] - the stack list Stack/Sample attributes:
+* name: string - the stack name
+* displayName: string - the display name of devfile
+* description: string - the description of devfile
+* tags: string[] - the tags associated to devfile
+* icon: string - the devfile icon
+* language: string - the project language that is used in the devfile
+* git: *git - the information of remote repositories
+
+.extraDevfileEntries.yaml sample
+
+====
+----
+schemaVersion: 1.0.0
+samples:
+  - name: nodejs-basic
+    displayName: Basic NodeJS
+    description: A simple Hello World Node.js application
+    icon: https://github.com/maysunfaisal/node-bulletin-board-2/blob/main/nodejs-icon.png
+    tags: ["NodeJS", "Express"]
+    projectType: nodejs
+    language: nodejs
+    git:
+      remotes:
+        origin: https://github.com/redhat-developer/devfile-sample.git
+  - name: code-with-quarkus
+    displayName: Basic Quarkus
+    description: A simple Hello World Java application using Quarkus
+    icon: .devfile/icon/quarkus.png
+    tags: ["Java", "Quarkus"]
+    projectType: quarkus
+    language: java
+    git:
+      remotes:
+        origin: https://github.com/elsony/devfile-sample-code-with-quarkus.git
+  - name: java-springboot-basic
+    displayName: Basic Spring Boot
+    description: A simple Hello World Java Spring Boot application using Maven
+    icon: .devfile/icon/spring-logo.png
+    tags: ["Java", "Spring"]
+    projectType: springboot
+    language: java
+    git:
+      remotes:
+        origin: https://github.com/elsony/devfile-sample-java-springboot-basic.git
+  - name: python-basic
+    displayName: Basic Python
+    description: A simple Hello World application using Python
+    icon: .devfile/icon/python.png
+    tags: ["Python"]
+    projectType: python
+    language: python
+    git:
+      remotes:
+        origin: https://github.com/elsony/devfile-sample-python-basic.git
+----
+====
+
+
+The file, index.json, contains the metadata of the stacks and samples in the registry. This metadata is also known as the devfile index schema. From the devfile index schema, you can get the stack and sample generic information such as: name, language, and resources.
```

Comment:
```adoc
The `index.json` file contains metadata for the stacks and samples in the registry.
```

I also don't believe we need the colon before listing "name, language, and resources."